configs/components/hiera.json

```diff
@@ -1 +1 @@
-{"url": "git://github.com/puppetlabs/hiera.git", "ref": "3047016da65d12ff7e316db17e40adec9a080090"}
+{"url": "git://github.com/puppetlabs/hiera.git", "ref": "refs/tags/3.1.0"}
```

Comment:
oops! I had accidentally fetched the facter repo in my hiera repo! Thanks, Kylo! With that fixed, the 3.1.0 tag does correspond with the `3047016da65d12ff7e316db17e40adec9a080090` sha :) yay!